src/engine/dict/src/math_symbol_key.rs

```diff
@@ -0,0 +1,32 @@
+use std::ops::{BitOr, BitOrAssign};
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Copy)]
+pub struct Style(pub u8);
```

Comment:
내부 필드를 노출시킬 이유가 없는것 같습니다.

```suggestion
pub struct Style(u8);
```

Reply:
모든 위치에서 Style이 노출되서 별로긴 한데, 저걸 숨기면 코드제너레이된 파일에서 Style 생성자를 못씁니다. 다른 방법이 있나요?

Reply:
코드생성하실때 Debug 구현을 사용하지말고 `Style::XXX | Stlye::YYY `로 하실순 없나요?

Reply:
`bitor`가 const fn이 아니기 때문에 static 변수 초기화에 문제가 생기네요